recipes/oset/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "oset" %}
+{% set version = "0.1.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4c1fd7dec96eeff9d3260995a8e37f9f415d0bdb79975f57824e68716ac8f904" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - oset
+
+about:
+  home: https://gitorious.com/sleipnir/python-oset
+  license: PSF
```

Comment:
I have seen it in a number of other recipes, so I guess it's fine. ping @conda-forge/core